pytorch_lightning/trainer/argparse_connector.py

```diff
@@ -0,0 +1,134 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, List, Tuple, Union
+from pytorch_lightning.utilities import argparse_utils
+from argparse import ArgumentParser, Namespace
+
+
+class ArgparseClass(object):
```

Comment:
what is the reason for this wrapper as it just calls other functions?

Reply:
we have 3 different implementations of this. we want to centralize it so they can all share the same one eventually.
second, the trainer.py file needs to be just the train code if possible. it's littered with random code that has nothing to do with training... (argparse has nothing to do with training)

Reply:
so this is the initial step of further refactoring, right?